Open_Science_New.html

```diff
@@ -291,83 +346,80 @@ <h2>Standardization</h2>
 	  </section>
-	
+
 	  <section>
 	    <h2>SQL versus NoSQL</h2>
 	    <img src="images/NoSQL_database.jpg" width="77%">
 	  </section>
-	
-	  <!-- Git and Github -->
-	  <section data-transition="fade">
-	    <table class="reveal">
-	      <h2>Git & Github</h2>
-	      <tr>
-		<td width="50%" style="vertical-align: middle">
-		  <img src="images/git.png">
-		</td>
-		<td width="50%" style="vertical-align: middle">
-		  <img src="images/Github.png">
-		</td>
-	    </tr></table>
-	  </section>
-	  
-	  <!-- Git Annex -->
-	  <section data-transition="fade">
-	    <img src="images/Git_annex.png">
-	  </section>
 
-	  <!-- Git Annex -->
-	  <section data-transition="fade">
-	    <img src="images/Datalad.png">
-	  </section>
+	  <!-- Git -->
+              <section>
+                  <h3>Version Control</h3>
+               <table class="reveal">
+		<tr>
+		  <td width="40%" height="40%">
+                    <a href="https://git-scm.com/"><img src="images/git.png"></a>
+		  </td>
+            <td width="40%" height="40%">
+                    <a href="https://github.com/"><img src="images/Github.png"></a>
+		  </td>
+		</tr>
+                   <tr>
+                   <td width="40%" height="40%">
+                    <a href="https://git-annex.branchable.com/"><img src="images/Git_annex.png"></a>
+		  </td>
+            <td width="40%" height="40%">
+                    <a href="https://www.datalad.org/"><img src="images/Datalad.png"></a>
+		  </td>
+                   </tr>
+              </table>
+              </section>
 	</section>
 
-
-	 <!-- Other considerations -->
-
-	  <!-- Centralized versus De-centralized -->
-	  <section data-transition="concave">
-	    <h2>Centralized or Distributed?</h2>
-	    <img src="images/Networks.png" width="80%">
-	  </section>
-
-	  
-	  <!-- Ethics Framework -->
+        <!-- Ethics Framework -->
 	  <section>
 	    <section>
-	      <h2>Ethics Framework</h2>
-	      <img width="80%" src="images/Consent_codes.png">
+             <h2>Ethics Framework</h2>
+             <table class="reveal">
+		<tr>
+		  <td width="50%" style="vertical-align: middle">
+		    <ul><li>A systematic way of recording data</li>
+                <li>Uses condition based on consent permissions</li>
+                <li>Found in the datasets of the main public genome archives</li></ul>
+		  </td>
+		  <td>
+		    <img src="images/Consent_codes.png">
+	      </td></tr></table>
 	    </section>
-	    
+
 	    <!-- Problem Case -->
 	    <section data-transition="convex">
 	      <h2>Privacy Concerns</h2>
 	      <table class="reveal">
 		<tr>
 		  <td width="40%" style="vertical-align: middle">
-		    <img src="images/Data_breach_title.png">
+		    <a href="https://www.wired.com/2016/05/okcupid-study-reveals-perils-big-data-science/"><img src="images/Data_breach_title.png">
 		    <img src="images/Data_breach_consent.png">
-		    <img src="images/Data_breach.png">
-		    
+		    <img src="images/Data_breach.png"></a>
```

Comment:
@christinerogers Another fitting issue :( Also - do I need to add contextual text here? the articles and headlines seem self-explanatory

Reply:
https://htmlpreview.github.io/?https://github.com/knoll-alex/samirdas.github.io/blob/ak-Open_Science/Open_Science_New.html#/10/1